letsencrypt/storage.py

```diff
@@ -136,14 +141,17 @@ def _consistent(self):
 
         """
         # Each element must be referenced with an absolute path
-        if any(not os.path.isabs(x) for x in
-               (self.cert, self.privkey, self.chain, self.fullchain)):
-            return False
+        for x in (self.cert, self.privkey, self.chain, self.fullchain):
+            if not os.path.isabs(x):
+                logger.debug("Element %s is not referenced with an "
+                             "absolute file.", x)
```

Comment:
nit: I think "Element %s is not referenced with an absolute **path**" is more accurate.